example/Makefile

```diff
@@ -13,7 +13,7 @@ endif
 
 # Run against the configured Kubernetes cluster in ~/.kube/config
 run:
-	go run ../main.go
+	go run ../main.go run
 
 # Install CRDs into a cluster
 install: kustomize
```

Comment:
IMO we need to add the kustomize as pre-requirement and address it via docs at least as first moment.